compiler/rustc_mir_transform/src/shim.rs

```diff
@@ -154,6 +154,7 @@ fn make_shim<'tcx>(tcx: TyCtxt<'tcx>, instance: ty::InstanceDef<'tcx>) -> Body<'
             &add_moves_for_packed_drops::AddMovesForPackedDrops,
             &deref_separator::Derefer,
             &remove_noop_landing_pads::RemoveNoopLandingPads,
+            &lower_intrinsics::LowerIntrinsics,
```

Comment:
Why was this added?

Reply:
Actually, I know why. I wonder if it's better to place this in `build_call_shim`, or even make our own function like `build_reified_intrinsic`.